manifests/config/file.pp

```diff
@@ -0,0 +1,24 @@
+# == Define: supervisord::config::file
+#
+define supervisord::config::file (
+  $command          => undef,
+  $directory        => undef,
+  $user             => undef,
+  $process_name     => '%(program_name)s_%(process_num)s',
+  $startsecs        => 10,
+  $numprocs         => 1,
+  $autostart        => 'true',
+  $autorestart      => 'true',
+  $redirect_stderr  => 'true',
+  $startretries     => 3,
+) {
+  file { "${::supervisord::params::includedir}/${title}.conf":
```

Comment:
ah, I see what you mean now. In that case, I think the files defined through puppet won't work, as the parameter `${::supervisord::params::includedir}` will be always the default supervisord folder.